### ISSUE TITLE
Update Header and navigation bars to make them mobile ready

### ISSUE THREAD

**Issue by briri** [CDLUC3/dmptool#137]
The current HTML layout of the navigation bars is incorrect and prevents the default Bootstrap hamburger icon from appearing when viewed on mobile devices.

You can simulate the mobile view by zooming in on your browser.

Refer to the Bootstrap documentation for the correct HTML layout to use. Take note of the order in which the classes are used (e.g. row, navbar, etc.): https://getbootstrap.com/docs/3.3/components/#navbar

**Comment by briri** [comment]
Made some additional modifications based on feedback from @rhu001.

The site should be more responsive now ... as far as the header and footer are concerned (there are still some mobile display issues with the main content areas. Changing this would either require extensive customizations or changes to the core DMPRoadmap codebase). 

All of the menus (fixed sign/signout, language, learn, create plan, dashboard, admin and Org specific links and email) are folded into the main hamburger menu when viewed from a mobile device. The Org name or the DMPTool logo appears to the left of the hamburger menu depending on whether the user is logged in and they are affiliated with an Org.

@stephaniesimms and @rhu001 if you could give it another review on dmptool-stg (especially from a tablet device) I would appreciate it. Please leave any suggestions for changes on this ticket.

**Comment by stephaniesimms** [comment]
i did some testing through resizing desktop browser windows and on mobile and noticed one minor thing:
- vertical spacing for the Learn menu items on prod fills the dropdown area evenly (screenshot 1); it needs adjustment on stage (screenshot 2)

dmptool prod
![spacing](https://user-images.githubusercontent.com/12703638/45717660-813eff00-bb4f-11e8-86f0-9fd5145bcefb.png)

dmptool stage
![spacing-fix](https://user-images.githubusercontent.com/12703638/45717671-8734e000-bb4f-11e8-90cb-0f0d3dd394e1.png)

**Comment by rhu001** [comment]
Tested in portrait view and landscape views on tablet. Landscape view showed normal desktop drop down menus; but, the portrait view did not show hamburger menu in right top corner. See attached screenshot below.

![69590c5e-936c-4a05-bcd2-efcd013baa0e](https://user-images.githubusercontent.com/23642279/45979200-33b90b00-c003-11e8-8e12-0dc00d6b9e58.png)

**Comment by rhu001** [comment]
Also, tested on Chrome browser on tablet and when trying to login via homepage the portrait view also failed to show hamburger menu in right corner and the text banner on homepage lost its background. See attached.

![faf2ecd6-abaa-4ab6-844c-21fe194c59cc](https://user-images.githubusercontent.com/23642279/45979389-ceb1e500-c003-11e8-876d-edeacbd7cf79.png)

**Comment by rhu001** [comment]
There was also a spacing issue that I noticed on the homepage in both desktop as well as tablet landscape view. Extra spacing in between DMPTool by the Numbers and Top 5 Templates.

![af3cbf5b-7a76-410c-8dab-1286faf0b3e4](https://user-images.githubusercontent.com/23642279/45980652-636a1200-c007-11e8-9578-fef888cc72aa.png)

**Comment by briri** [comment]
@stephaniesimms the link menu spacing should be better now.

@rhu001 I met with Joel and got some advice for my media queries and testing. Please take a look when you have a chanced let me know if things are ok on your iPad now.

**Comment by stephaniesimms** [comment]
link menu spacing is still off. most of the other things in this issue have been addressed so i'll close it and create a new issue with itemized fixes for spacing, etc.
